docker-compose.yaml

```diff
@@ -5,4 +5,6 @@ include:
   - ./services/backendnext/docker-compose.yaml
   - ./services/frontend/docker-compose.yaml
   - ./services/searchapi/docker-compose.yaml
+  - ./services/elastic/docker-compose.yaml
```

Comment:
can we move this to the backendnext docker-compose, as it is dedicated to it?

Reply:
sure